### PR TITLE
add invertLED command line option: Moducops LED is inverted

### PR DESCRIPTION
**Pull request by KlausPopp**

### CODE REVIEW

File: src/internal/devicestate/ledservice.go [hunk 1]
```diff
@@ -156,3 +150,14 @@ func (l *LedService) controlLed() {
 		}
 	}
 }
+
+func (l *LedService) SetLED(ledVal int) {
```
Comment (siredmar) [inline src/internal/devicestate/ledservice.go]: Don't exit from inside a method. Make a return value of type `error` and let the caller handle the error and panic if needed.

Comment (KlausPopp) [reply src/internal/devicestate/ledservice.go]: You may be right. But this wasn't introduced by my change.

This was the old code
`		ledVal := steps[curPattern][stepIdx]
		err := l.line.SetValue(ledVal)
		if err != nil {
			log.Println(err)
			os.Exit(1)
		}`

Comment (siredmar) [reply src/internal/devicestate/ledservice.go]: It was introduced by your change. However, don't you want to leave the code better as it was before?

Comment (KlausPopp) [reply src/internal/devicestate/ledservice.go]: Ok , fixed

**Comment by KlausPopp** [comment]
Replace by PR #4

**Pull request merged**